<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainBankGUI</class>
 <widget class="QDialog" name="MainBankGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color: rgb(255, 178, 25)
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: black;
}</string>
   </property>
   <property name="text">
    <string>Bank Management System</string>
   </property>
  </widget>
  <widget class="QLabel" name="BalanceLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: black;
}</string>
   </property>
   <property name="text">
    <string>Balance:  </string>
   </property>
  </widget>
  <widget class="QLabel" name="AccNumLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: black;
}</string>
   </property>
   <property name="text">
    <string>Account Number: </string>
   </property>
  </widget>
  <widget class="QLabel" name="UsernamLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: black;
}</string>
   </property>
   <property name="text">
    <string>Username:  </string>
   </property>
  </widget>
  <widget class="QPushButton" name="ViewFeaturesBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>310</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: black;
	background-color:white
}</string>
   </property>
   <property name="text">
    <string>Features</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LoanBtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>310</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: black;
	background-color:white
}</string>
   </property>
   <property name="text">
    <string>Loan Management</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ViewRprtBtn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: black;
	background-color:white
}</string>
   </property>
   <property name="text">
    <string>View Reports</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AccSettingBtn">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>80</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: black;
	background-color:white
}</string>
   </property>
   <property name="text">
    <string>Account Setting</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AdminPanelBtn">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>120</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: black;
	background-color:white
}</string>
   </property>
   <property name="text">
    <string>Admin Panel</string>
   </property>
  </widget>
  <widget class="QLabel" name="AnnounceLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>441</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: red;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
